flink-runtime/src/main/java/org/apache/flink/runtime/operators/DataSourceTask.java

```diff
@@ -165,6 +172,7 @@ public void invoke() throws Exception {
 					// close. We close here such that a regular close throwing an exception marks a task as failed.
 					format.close();
 				}
+				splitCounter.inc();
```

Comment:
Should we make the semantic of this counter more explicit and call it `completedSplitCounter`?